src/data.toml

```diff
@@ -1,18 +1,22 @@
 custom_fonts = "Fonts are experimental and some packs may break the application. You can get font packs listed here: https://discord.com/channels/1196075698301968455/1225584192012947506/1241453802184380418"
 ios_notifications = "For notifications to work on a Jailed device you need to sign the ipa with a certificate that has the notification entitlement enabled.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64` to check the architecture of your CPU and operating system."
 broken_addons = "You can find a curated list of broken add-ons in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
 altstore_pal = "AltStore PAL is an alternate app marketplace for notarized applications. You cannot use it to sideload.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 no_passkeys_when_pre_patched = "Passkeys are not supported when using the pre-patched application. Please use a different method to log in to your account or use the LSPosed module / iOS tweak."
-ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead (which generally uses an explicit App ID by default) if you want this functionality.\n\nKeep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-ios_file_access = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively disables the file access capability.\nYou need to sign the application with a local dev cert instead (which generally uses an explicit App ID by default) if you want this functionality.\n\nKeep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev CERT instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
+addon_404 = "You are not supposed to visit the URL directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
```

Comment:
```suggestion
addon_404 = "Please do not visit the URL directly since it won't display anything useful. Instead, copy the link and use it to install the add-on."
```